site/src/pages/api/icon-nodes/index.tsx

```diff
@@ -0,0 +1,10 @@
+import {getAllData} from '../../../lib/icons';
+import { parseSync } from 'svgson';
```

Comment:
`svgson` is not in the `package.json`

Reply:
I don't think it needs to be with pnpm and the workspace package.json: https://github.com/lucide-icons/lucide/blob/main/package.json#L42

Feel free to add it though.